client/state/editor/test/actions.js

```diff
@@ -1,21 +1,21 @@
 /**
  * External dependencies
  */
-import { expect } from 'chai';
 import { forEach } from 'lodash';
 
 /**
  * Internal dependencies
  */
-import { MODAL_VIEW_STAT_MAPPING, setEditorMediaModalView } from '../actions';
+import { MODAL_VIEW_STATS, setEditorMediaModalView } from '../actions';
```

Comment:
This test never worked, `MODAL_VIEW_STAT_MAPPING` was never defined in `../actions`. It was passing because it ends up being a `test` with no `expects`.